build/commands/lib/config.js

```diff
@@ -650,10 +650,8 @@ Config.prototype.update = function (options) {
     this.is_asan = false
   }
 
-  if (options.use_goma && options.use_goma !== false) {
-    this.use_goma = true
-  } else {
-    this.use_goma = false
+  if (options.use_goma !== undefined) {
+    this.use_goma = options.use_goma
```

Comment:
does this still work with`--use_goma` with no true or false?

Reply:
> does this still work with`--use_goma` with no true or false?

yes.

Reply:
I think maybe what we want here is `options.use_goma !== undefined && options.use_goma !== false`?